pkg/daemon/ceph/client/osd.go

```diff
@@ -355,3 +355,13 @@ func OSDOkToStop(context *clusterd.Context, clusterInfo *ClusterInfo, osdID, max
 
 	return stats.OSDs, nil
 }
+
+// OsdPrimaryAffinity assign primary-affinity (within range [0.0, 1.0]) to specific OSD.
```

Comment:
```suggestion
// OsdPrimaryAffinity assigns primary-affinity (within range [0.0, 1.0]) to a specific OSD.
```

Reply:
@sp98 To the best of my understanding, primary affinity is enabled by default; I believe the docs you are referring to are out-dated. Thanks for the nits